traffic_monitor/tests/integration/Dockerfile_run.sh

```diff
@@ -0,0 +1,394 @@
+      "fqdn": "server0.monitor-integration.test",
+      "hashCount": 1,
+      "hashId": "server0",
+      "httpsPort" : null,
+      "ip": "testcaches",
+      "ip6": null,
+      "locationId": "",
+      "port" : 30000,
+      "status": "REPORTED",
+      "type": "EDGE",
+      "deliveryServices": {"ds0":["ds0.monitor-integration.test"]},
+      "routingDisabled": 0
+      },
+  	"server1": {
+      "cacheGroup": "cg0",
+      "profile": "Edge0",
+      "fqdn": "server1.monitor-integration.test",
+      "hashCount": 1,
+      "hashId": "server1",
+      "httpsPort" : null,
+      "ip": "testcaches",
+      "ip6": null,
+      "locationId": "",
+      "port" : 30001,
+      "status": "REPORTED",
+      "type": "EDGE",
+      "deliveryServices": {"ds0":["ds0.monitor-integration.test"]},
+      "routingDisabled": 0
+      }
+	},
+	"deliveryServices": {
+    "ds0": {
+      "anonymousBlockingEnabled": false,
+      "consistentHashQueryParams": [],
+      "consistentHashRegex": "",
+      "coverageZoneOnly": false,
+      "dispersion": {
+			  "limit": 1,
+				"shuffled": false
+			},
+      "domains": ["ds0.monitor-integration.test"],
+      "geolocationProvider": null,
+      "matchsets": [
+			  {
+				  "protocol": "HTTP",
+					"matchlist": [
+					  {
+						  "regex": "\\.*ds0\\.*",
+							"match-type": "regex"
+						}
+					]
+				}
+			],
+      "missLocation": {"lat": 0, "lon": 0},
+      "protocol": {
+        "acceptHttp": true,
+        "acceptHttps": false,
+        "redirectToHttps": false
+      },
+      "regionalGeoBlocking": "false",
+      "responseHeaders": {},
+      "requestHeaders": [],
+      "soa": {
+        "admin": "60",
+        "expire": "60",
+        "minimum": "60",
+        "refresh": "60",
+        "retry": "60"
+			},
+      "sslEnabled": false,
+      "ttl": 60,
+      "ttls": {
+        "A": "60",
+        "AAAA": "60",
+        "DNSKEY": "60",
+        "DS": "60",
+        "NS": "60",
+        "SOA": "60"
+			},
+      "maxDnsIpsForLocation": 3,
+      "ip6RoutingEnabled": false,
+      "routingName": "ccr",
+      "bypassDestination": null,
+      "deepCachingType": null,
+      "geoEnabled": false,
+      "geoLimitRedirectURL": null,
+      "staticDnsEntries": []
+    }
+	},
+	"edgeLocations": {
+	  "cg0": {"latitude":0, "longitude":0}
+	},
+	"trafficRouterLocations": {
+	  "tr0": {"latitude":0, "longitude":0}
+	},
+	"monitors": {
+    "trafficmonitor": {
+        "fqdn": "trafficmonitor.monitor-integration.test",
+        "httpsPort": null,
+        "ip": "trafficmonitor",
+        "ip6": null,
+        "location": "cg0",
+        "port": 80,
+        "profile": "Monitor0",
+        "status": "REPORTED"
+    }
+	},
+	"stats": {
+    "CDN_name": "fake",
+    "date": 1561000000,
+    "tm_host": "testto",
+    "tm_path": "/fake",
+    "tm_user": "fake",
+    "tm_version": "integrationtest/0.fake"
+	}
+}
+'
+
+	curl -Lvsk ${TESTTO_URI}/api/1.2/cdns/fake/configs/monitoring.json -X POST -d '
```

Comment:
As with the rest of the API v2.0 routes, this has become `cdns/fake/configs/monitoring`, `json` suffix no longer supported